src/index.js

```diff
@@ -2,7 +2,7 @@ const webpack = require('webpack')
 const CommonJsRequireDependency = require('webpack/lib/dependencies/CommonJsRequireDependency')
 const { createFsFromVolume, Volume } = require('memfs')
 const { sep, dirname, relative } = require('path')
-const { cache } = require('webpack')
+// const { cache } = require('webpack') // Cache handling missing
```

Comment:
why is this commented out?

Reply:
As I said in my commit, I couldn't find where to handle the cache in the refactoring: https://github.com/pinojs/pino-webpack-plugin/pull/59#issuecomment-1386734928
To be able to emit assets from the child compilation, I had to use different webpack hooks...

Reply:
Actually, it's nearly a complete refactoring, I thought it would be more readable this way:
- I got into the (too light) Webpack hooks documentation, and used more appropriate hooks to handle additional assets
- I thought that having functions declared further in code but used only once doesn't help readability. I used the function body directly in the webpack hooks

Please let me know if you think I should have kept the previous code "style"...